components/webgpu/lib.rs

```diff
@@ -414,6 +422,17 @@ impl WGPU {
                         )
                     }
                 },
+                WebGPURequest::CreateSampler {
+                    device_id,
+                    sampler_id,
+                    descriptor,
+                } => {
+                    let global = &self.global;
+                    let desc = descriptor.map_label(move |l| {
+                        CString::new(l.as_bytes()).unwrap().as_c_str().as_ptr()
```

Comment:
this looks wrong: the `CString` will be dropped before the pointer is used.
You can use something [like this](https://github.com/gfx-rs/wgpu/blob/70154373f8668b88c43fcbdee7bee8a89816eb39/player/src/main.rs#L242-L244) instead.